web-registry/src/lib/ledger.ts

```diff
@@ -2,6 +2,7 @@ import getApiUri from './apiUri';
 
 export const chainId = process.env.REACT_APP_LEDGER_CHAIN_ID;
 export const nctBasket = process.env.REACT_APP_NCT_BASKET === 'true';
+export const isBridgeFlag = process.env.REACT_APP_BRIDGE === 'true';
```

Comment:
it doesn't really matter, but I think I would have called this flag `isBridgeEnabled` instead

Reply:
Agreed